### DISCUSSION TITLE
LVMError: pvcreate failed for /dev/sda3: running

### DISCUSSION THREAD

**Discussion by vdr0011** [Security-Onion-Solutions/securityonion#10035] [General]
I keep receiving this error when trying to install security onion vm workstation 17
![Screenshot 2023-03-28 101921](https://user-images.githubusercontent.com/128842871/228290123-af97e3ed-2c7d-4afb-b6e2-40b677ce494a.png)

**Comment by cm-ops** [comment]
Was there a previous LVM on the server?

**Reply by vdr0011** [reply]
Not sure what you mean. I was just trying to install the SecOnion ISO on my VM Workstation Pro, based on the instructions on the SecOnion site, using Centos 7 64-bit

**Reply by cm-ops** [reply]
Meaning was this a reinstall? How did you set up the storage for the VM (one HDD or multiple)?
